sdk/src/utils/public/types.ts

```diff
@@ -0,0 +1,9 @@
+export enum SwapDirection {
+  AtoB = "aToB",
+  BtoA = "bToA",
+}
+
+export enum TokenType {
+  TokenA = 1,
+  TokenB,
```

Comment:
Shoudl this be `TokenB = 2,` ?

Reply:
Typescript will automatically fill that in if you specify the starting number